manifest.toml

```diff
@@ -0,0 +1,55 @@
+packaging_format = 2
+
+id = "pixelfed"
+name = "Pixelfed"
+description.en = "ActivityPub Federated Image Sharing"
+description.fr = "Logiciel de partage d'image fédéré via ActivityPub"
+
+version = "0.11.7~ynh1"
+
+maintainers = ["yalh76", "lapineige"]
+
+[upstream]
+license = "AGPL-3.0-or-later"
+website = "https://pixelfed.org/"
+admindoc = "https://docs.pixelfed.org/running-pixelfed/administration.html"
+userdoc = "https://docs.pixelfed.org/"
+code = "https://github.com/pixelfed/pixelfed"
+
+[integration]
+yunohost = ">= 11.1.19"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+[resources]
+        [resources.sources.main]
+        url = "https://github.com/pixelfed/pixelfed/archive/refs/tags/v0.11.7.tar.gz"
+        sha256 = "eb27aa61c337c3dbca3c0e59cbd86069da1dbc7dea988630d3453cd92dcc75e7"
+        autoupdate.strategy = "latest_github_tag"
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "postgresql libgd3 jpegoptim optipng pngquant ffmpeg imagemagick supervisor php8.1-redis php8.1-bcmath php8.1-cli php8.1-ctype php8.1-curl php8.1-exif php8.1-gd php8.1-iconv php8.1-intl php8.1-mbstring php8.1-tokenizer php8.1-xml php8.1-zip php8.1-pdo php8.1-pgsql php8.1-fileinfo php8.1-imagick"
```

Comment:
So now instead of a single "PHP version" we will have to change all those values ? :(

Reply:
yes